### PR TITLE
Mention that brackets are redundant if using a parameterless generic …

### PR DESCRIPTION
**Pull request by SteveDunn**
…attribute

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/csharp/whats-new/csharp-11.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ public string Method() => default;
 ```
 
 You must supply all type parameters when you apply the attribute. In other words, the generic type must be [fully constructed](~/_csharpstandard/standard/types.md#84-constructed-types).
+In the example above, the empty brackets can be ommitted as the attribute does not have any arguments.
```
Comment (BillWagner) [inline docs/csharp/whats-new/csharp-11.md:76]: Let's clarify the characters that can be omitted:

```suggestion
In the example above, the empty parentheses (`(` and `)`) can be ommitted as the attribute does not have any arguments.
```

**Review by BillWagner — COMMENTED** [review]
This is a great addition @SteveDunn 

I had one comment to clarify, so that readers don't mistakenly think the `[` and `]` could be removed.

After that, it's ready to merge.

**Comment by SteveDunn** [comment]
Looks better - thanks @BillWagner

**Review by BillWagner — APPROVED** [review]
Thanks @SteveDunn 

This looks great. I'll :shipit: now.

**Pull request merged**